src/controllers/users.controller.js

```diff
@@ -0,0 +1,97 @@
+const { userSchema } = require('../shemas');
```

Comment:
There is a typo in the import path for `userSchema`. It should be `../schemas` instead of `../shemas` to match the correct directory name.